install-3scale-rwo/apimanager.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: apps.3scale.net/v1alpha1
+kind: APIManager
+metadata:
+  name: "3scale-amp"
+  namespace: "3scale-rwo"
+spec:
+  wildcardDomain: apps.cluster-0228.0228.sandbox1124.opentlc.com
```

Comment:
this exact domain won't work for someone else, so it's normally better to leave this as a variable or note so they know to replace it

Reply:
This does not belong here. Removed it